README.md

```diff
@@ -7,7 +7,7 @@ Bi-directional codecs for [argonaut](https://github.com/purescript-contrib/pures
 
 This library is build on [`purescript-codec`](https://github.com/garyb/purescript-codec) and offers a different approach to dealing with JSON encoding/decoding than [`purescript-argonaut-codecs`](https://github.com/purescript-contrib/purescript-argonaut-codecs). Instead of using type classes, codecs are constructed as values explicitly. As long as the basic codec values provided by this library are used, the codecs are guaranteed to roundtrip successfully.
 
-The errors reported from this library are a little better than those provided by `purescript-argonaut-codecs` too - they contain the full JSON structure to the point of failure, and the error can be inspected as a value before being printed as a string.
+The errors reported from this library are a little better than those provided by `purescript-argonaut-codecs` too - when used with `showJsonDecodeError` they show a helpful explanation of what when wrong and where.
```

Comment:
`what when wrong` sh be
`what went wrong`

Reply:
Fixed.